unix-rpm/src/main/java/com/stratio/mojo/unix/rpm/SpecFile.java

```diff
@@ -179,6 +179,24 @@ public void streamTo( LineStreamWriter spec )
             spec.add( "%define " + defineStatement );
         }
 
+        // Ref: http://blog.agilepartner.net/building-rpm-packages-2/         
```

Comment:
You can let this url commented in the code and every other blog content as this PR comment.

This way, the PR will just contain 3 lines (at this class)